test/controllers/unattended_controller_test.rb

```diff
@@ -237,7 +237,6 @@ class UnattendedControllerTest < ActionController::TestCase
       ptable_ubuntu = FactoryBot.create(:ptable, :ubuntu, :name => 'ubuntu default',
         :layout => 'd-i partman-auto/disk string /dev/sda\nd-i partman-auto/method string regular...',
                                          :operatingsystem_ids => [operatingsystems(:ubuntu1010).id])
-      # MySQL datetime precision is seconds - explicitly create host1 with tomorrow's created_at
```

Comment:
I'd keep this in, perhaps in a different way:
```
# explicitly create host1 with tomorrow's created_at so it's guaranteed to be newer than rh_host
```

Reply:
updated